config/routes.rb

```diff
@@ -30,8 +30,9 @@
   get "/upvoted", to: redirect("/upvoted/stories")
   get "/upvoted/page/:page", to: redirect("/upvoted/stories/page/%{page}")
 
-  get "/top(/:length(/page/:page))/rss" => "home#top", :format => "rss"
+  get "/top/rss" => "home#top", :format => "rss"
   get "/top(/:length(/page/:page))" => "home#top", :as => "top"
+  get "/top(/:length(/page/:page))/rss" => "home#top", :format => "rss"
```

Comment:
```suggestion
  get "/top(/:length)/rss" => "home#top", :format => "rss"
```

I don't want to add an rss url for each page.